src/pydvl/valuation/types.py

```diff
@@ -52,6 +53,16 @@ def __hash__(self):
         sha256_hash = hashlib.sha256(self.subset.tobytes()).hexdigest()
         return int(sha256_hash, base=16)
 
+    def with_idx_in_subset(self) -> Self:
```

Comment:
I think this method should get a small docstring, in particular mentioning the error raise in case the index is not set.

Reply:
Agree, done.